deployments/helm/sumologic-the-coffee-bar/values.yaml

```diff
@@ -114,18 +116,72 @@ resources:
   calculatorsvc:
     common:
       requests:
-        cpu: 100m
-        memory: 50Mi
+        cpu: 150m
+        memory: 200Mi
       limits:
-        cpu: 250m
+        cpu: 300m
         memory: 500Mi
   clicker:
     requests:
-      cpu: 200m
+      cpu: 250m
       memory: 512Mi
     limits:
       cpu: 500m
       memory: 1024Mi
+  # specifying postgres pod resource limits
```

Comment:
Please decrease the requests and limits to `common` values. There's no need to increase limits for basic usage.